### ISSUE TITLE
Loading .vec Files and Difference Between Embeddings

### ISSUE THREAD

**Issue by Hellgasm** [facebookresearch/fastText#722]
Hello,
I have two problems that I cannot find answer to in the documentation.

1. How can I load .vec file (pretrained or generated by Fasttext) to Fasttext to actually use it. I haven't downloaded the binary model. Is it possible?

2. What difference are there between the word embeddings created using supervised and other methods(cbow/skipgram)?

**Comment by fclesio** [comment]
@Hellgasm 

For training a new model in Python if you have the `.vec` file you do not need the `.bin` generated in the embedding training. The only thing that you need it's to change the parameter `pretrainedVectors`

Example in Python:
```
model = train_supervised(input=train_data,
                         epoch=epoch_val,
                         lr=lr_val,
                         wordNgrams=wordNgrams_val,
                         verbose=2,
                         minCount=3000,
                         pretrainedVectors='my_embeddings_corpora.vec'
                        )
```

The parameter in command line [you can find here in docs](https://fasttext.cc/docs/en/options.html).

**Comment by sid-sundrani** [comment]
Yes, this seems to be a workaround for loading .vec files in fasttext instead of gensim.

one advantage of loading it in fasttext is-
1.  uses lower memory than genism (got OOM when loading with gensim) 
2. you can get word vectors of unseen works 

```
with open('test.txt', 'w', encoding='utf-8') as f:
    f.write('jack')

model = fasttext.train_supervised('test.txt',
                            epoch=0,
                            pretrainedVectors='wiki-news-300d-1M.vec',
                            dim=300
                        )
```
